posts/bksahus-gsoc-wrap-up.rst

```diff
@@ -0,0 +1,166 @@
+Hello everyone!
+
+GSoC 2019 has almost come to an end! It's the time to wrap up this mega event
+started back in May 2019. Under the mentorship of Mentor Hayen, my learning
+experience has undergone a roller-coaster ride and it has not only boosted my
+growth as a developer but also as an individual. Over the last 3 months the
+followings are my major contributions to this project:
+
+
+Built-ins Optimizations
+=======================
+
+* "any": `PR #246 <https://github.com/Nuitka/Nuitka/pull/246>`
+    * `nuitka.nodes.BuiltinAnyNodes` node added to optimize the "any" built-in.
+    * Developed an algorithm to predict the "any" for arguments having repetitive values at compile time. 
+      For example::
+      
+         any([0]*255) -> False
+         any([False, False, True]) -> True
+         
+    * Extended support for `range`, `set` and `dict` built-ins.
+    * Added the optimized C side support too
+    * Added a method `getMetaClassBase` to make Python 2 and Python 3 compatible while working 
+    with metaclasses.
+         
+* Issue reported and closed `Issue #349 <https://github.com/Nuitka/Nuitka/issues/349>`
+    * Created a new module `nuitka.nodes.IterationHandles` to work with iterables.
+    * Added support of Iteration for non-mutable types.
+
+* "all": `PR #407 <https://github.com/Nuitka/Nuitka/pull/407>`
+    * Added `nuitka.nodes.BuiltinAllNodes` to optimize the "all" built-ins.
+    * Developed an algorithm similar to "any" to predict the "all" arguments.
+      For example::
+      
+        all([0, 0, 1]) -> False
```

Comment:
Is this not missign a `code-block` directive?

Reply:
It's already in a `code-block` directive